diff --git a/node_modules/@stryker-mutator/typescript-checker/dist/src/tsconfig-helpers.js b/node_modules/@stryker-mutator/typescript-checker/dist/src/tsconfig-helpers.js
index 4b4dd6e..8e72bd5 100644
--- a/node_modules/@stryker-mutator/typescript-checker/dist/src/tsconfig-helpers.js
+++ b/node_modules/@stryker-mutator/typescript-checker/dist/src/tsconfig-helpers.js
@@ -27,7 +27,7 @@ const LOW_EMIT_OPTIONS_FOR_PROJECT_REFERENCES = Object.freeze({
 });
 function guardTSVersion() {
     if (!semver_1.default.satisfies(typescript_1.default.version, '>=3.6')) {
-        throw new Error(`@stryker-mutator/typescript-checker only supports typescript@3.6 our higher. Found typescript@${typescript_1.default.version}`);
+        // throw new Error(`@stryker-mutator/typescript-checker only supports typescript@3.6 our higher. Found typescript@${typescript_1.default.version}`);
     }
 }
 exports.guardTSVersion = guardTSVersion;
